static/app/views/issueDetails/streamline/eventList.spec.tsx

```diff
@@ -0,0 +1,217 @@
+import {EventFixture} from 'sentry-fixture/event';
+import {EventsStatsFixture} from 'sentry-fixture/events';
+import {GroupFixture} from 'sentry-fixture/group';
+import {LocationFixture} from 'sentry-fixture/locationFixture';
+import {OrganizationFixture} from 'sentry-fixture/organization';
+import {ProjectFixture} from 'sentry-fixture/project';
+import {RouterFixture} from 'sentry-fixture/routerFixture';
+import {TagsFixture} from 'sentry-fixture/tags';
+
+import {render, renderHook, screen, userEvent} from 'sentry-test/reactTestingLibrary';
+
+import PageFiltersStore from 'sentry/stores/pageFiltersStore';
+import ProjectsStore from 'sentry/stores/projectsStore';
+import {useLocation} from 'sentry/utils/useLocation';
+import {useEventColumns} from 'sentry/views/issueDetails/allEventsTable';
+import {EventDetails} from 'sentry/views/issueDetails/streamline/eventDetails';
+import {MOCK_EVENTS_TABLE_DATA} from 'sentry/views/performance/transactionSummary/transactionEvents/testUtils';
+
+jest.mock('sentry/utils/useLocation');
+jest.mock('sentry/components/events/suspectCommits');
+jest.mock('sentry/views/issueDetails/groupEventDetails/groupEventDetailsContent');
+jest.mock('sentry/views/issueDetails/streamline/issueContent');
+jest.mock('screenfull', () => ({
+  enabled: true,
+  isFullscreen: false,
+  request: jest.fn(),
+  exit: jest.fn(),
+  on: jest.fn(),
+  off: jest.fn(),
+}));
+
+const mockUseNavigate = jest.fn();
```

Comment:
doesn't seem like this is being used

Reply:
Forgot to remove them when I moved the test to `eventDetails.spec.tsx`. Thanks 👍